docs/themes/avocadocs/layouts/partials/head/meta-tags.html

```diff
@@ -1,5 +1,8 @@
 <!--  Meta tags -->
 
+# Site verification
```

Comment:
```suggestion
<!-- Site verification -->
```

I don't think it's necessary in the final build. It also makes the HTML invalid.